src/loaders/lottie/tvgLottieModel.h

```diff
@@ -334,6 +334,148 @@ struct LottieMarker
     }
 };
 
+
+struct LottieTextFollowPath
+{
+    LottieFloat firstMargin = 0.0f;
+    LottieToggle perpendicular = 1;
```

Comment:
toggle has true/false
btw, this looks weird a bit. A keyframe toggle wouldn't be suitable for animation frames, breaking the smoothness.

Reply:
> toggle has true/false 

I know but I wanted to use one of the existing LottieGenericProperties instead of introducing it for bool.

> btw, this looks weird a bit. A keyframe toggle wouldn't be suitable for animation frames, breaking the smoothness.

agreed, but this is how it works in AE. we can solve both of these comments by storing the value as a regular bool. Do you think it's ok?
![Mar-11-2025 14-23-52](https://github.com/user-attachments/assets/9da0ff92-c7e8-4d4b-89e4-ea00be7e0449)

Reply:
shoot it please, `bool perpendicular = true;`